osu.Game.Tests/Database/BeatmapImporterTests.cs

```diff
@@ -1081,15 +1080,5 @@ private static void waitForOrAssert(Func<bool> result, string failureMessage, in
 
             Assert.Fail(failureMessage);
         }
-
-        public class NonOptimisedBeatmapImporter : BeatmapImporter
-        {
-            public NonOptimisedBeatmapImporter(RealmAccess realm, Storage storage)
-                : base(realm, storage)
-            {
-            }
-
-            protected override bool HasCustomHashFunction => true;
-        }
```

Comment:
Of note, this is no longer required because it was forcing off the "quick" import logic which is now implicitly off for single beatmap imports.